amethyst_ui/src/button/text_color.rs

```diff
@@ -0,0 +1,9 @@
+use amethyst_core::specs::prelude::{Component, DenseVecStorage};
+
+pub struct UiButtonTextColor {
+    pub text_color: [f32; 4],
```

Comment:
Technically this information is already available in the text entity, in the UiText component. Is this needed?

Reply:
The `UiButton` component had this information previously, so I moved it into its own component since the `UiButton` is now a `Widget` and not a `Component` anymore. I can remove it if it's not needed?